### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by LillyJekow**
- Updated all screenshots to remove the extension ID/Partner ID
- Updated screenshot lifecycle of an extension - development
- Deleted img for playground backoffice

Doc:
- index: Added terminology - developer portal
- Playground settings: filled out LumApps context details / Updated build tab
- Partner Organization: removed GitHub section + updated manage partner organization info
- Develop-extensions: removed manifests
- Extensions-test: Updated LumApps test info
- Manage-extensions: updated info to mention developer portal

### CODE REVIEW

File: docs/extensions/submit-extension.md [hunk 1]
```diff
@@ -14,51 +14,32 @@ To ease the review, you have to open a specific PR for each deployment request w
 
 <h6>Table of Contents</h6>
 
--   [Create your Pull Request](#create-your-pull-request)
-  -   [Pull Request labels](#pull-request-labels)
-  -   [Approval and rejection](#approval-and-rejection)
+-   [Submit your extension](#create-your-pull-request)
+-   [Pull Request labels](#pull-request-labels)
```
Comment (gregorycat) [inline docs/extensions/submit-extension.md:18]: This line can be removed

File: docs/extensions/submit-extension.md [hunk 2]
```diff
@@ -14,51 +14,32 @@ To ease the review, you have to open a specific PR for each deployment request w
...
-| deployed    | LumApps has deployed your extension. Customers can now use it in production.                                                 |
+The extension submission is done from the developer portal.
 
 ### Approval and rejection
 
 After an extension version is reviewed by LumApps, there are two possible outcomes:
 
--   **Approved** means your extension respects our guidelines. You can ask for production deployment for your extension. See [Extension Deployment](/submit-extension.md/extension_deployment).
--   **Rejected** means that your extension may not respect our guidelines and some revisions are required. The reason is provided in the PR. You can edit your extension, test it again, and resubmit your extension when you are ready.
+-   **Approved** means your extension respects our guidelines and you are allowed to deploy it.
```
Comment (gregorycat) [inline docs/extensions/submit-extension.md:31]: Once approved, the extension is automatically deployed

**Review by gregorycat — CHANGES_REQUESTED** [review]

File: docs/extensions/submit-extension.md (hunk 1 above)
Comment (LillyJekow) [reply docs/extensions/submit-extension.md:18]: done

File: docs/extensions/submit-extension.md (hunk 2 above)
Comment (LillyJekow) [reply docs/extensions/submit-extension.md:31]: done

**Review by gregorycat — DISMISSED** [review]

**Review by germainruget — DISMISSED** [review]

**Review by germainruget — APPROVED** [review]

**Pull request merged**